File: 01-basic-operations/node/deleteTable.js
const AWS = require("aws-sdk");
const DynamoDB = new AWS.DynamoDB({ region: "us-east-1" });

DynamoDB.deleteTable(
  {
    TableName: "Users",
  },
  function (err) {
    if (err) console.log(`Error deleting table: ${err}`);
    else console.log("Table deleted successfully! 🙌");
  }
);

File: 02-query/python/delete_table.py
import boto3

client = boto3.client("dynamodb", region_name="us-east-1")

try:
    client.delete_table(
        TableName="MovieRoles",
    )
    print("Table deleted successfully! 🙌")
except Exception as err:
    print(f"Error deleting table: {err}")
